10_code/fl_diff_in_diff_overdoses.ipynb

```diff
@@ -0,0 +1,1643 @@
+       "</script>"
+      ],
+      "text/plain": [
+       "alt.LayerChart(...)"
+      ]
+     },
+     "execution_count": 62,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "fit_other_states_before, reg_chart_other_states_before = get_reg_fit(\n",
+    "    other_states_before, yvar=\"Overdose_Per_100K\", xvar=\"Year\", color=\"red\", title= \"Overdoses per 100k Residents - Comp States\",alpha=0.05\n",
+    ")\n",
+    "\n",
+    "\n",
+    "fit_other_states_after, reg_chart_other_states_after = get_reg_fit(\n",
+    "    other_states_after, yvar=\"Overdose_Per_100K\", xvar=\"Year\", color=\"red\", title= \"Overdoses per 100k Residents - Comp States\",alpha=0.05\n",
+    ")\n",
+    "\n",
+    "(reg_chart_florida_before + reg_chart_florida_after + reg_chart_other_states_before + reg_chart_other_states_after + line).properties(width=700, height=500)\n"
+   ]
```

Comment:
Hey Annapurna, I surprisingly found that you have made those changes in your new ipynb files. Good job!